src/components/CalenderEntry.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+interface CalenderEntryProps {
+  name: string;
+  room: string;
+  date: string;
+  type: 'PRESENCE' | 'ONLINE';
+  startTime: string;
+  endTime: string;
+}
+
+function formatStartEndTime(startTime: string, endTime: string): string {
+  let newStartTime = new Date(startTime)
+    .toTimeString()
+    .split(' ')[0]
+    .slice(0, 5);
+  let newEndTime = new Date(endTime).toTimeString().split(' ')[0].slice(0, 5);
+
+  return `${newStartTime}-${newEndTime}`;
+}
+
+export default function CalenderEntry({
```

Comment:
*Calendar

Reply:
resolved in: https://github.com/stuv-mosbach/StuVReactNativeFrontend/commit/fd3538ed298358c905a167a5b704575e6b1118fb